workflows/rnaseq/rnaseq-standard.wdl

```diff
@@ -77,38 +81,33 @@ workflow rnaseq_standard {
     call picard.sort as picard_sort { input: bam=alignment.star_bam, max_retries=max_retries }
     call samtools.index as samtools_index { input: bam=picard_sort.sorted_bam, max_retries=max_retries }
     call picard.validate_bam { input: bam=picard_sort.sorted_bam, max_retries=max_retries }
-    call fqc.fastqc { input: bam=picard_sort.sorted_bam, max_retries=max_retries }
+    call qc.parse_validate_bam { input: in=validate_bam.out, max_retries=max_retries }
     call ngsderive.infer_strand as ngsderive_strandedness { input: bam=picard_sort.sorted_bam, bai=samtools_index.bai, gtf=gencode_gtf, max_retries=max_retries }
-    call qualimap.bamqc as qualimap_bamqc { input: bam=picard_sort.sorted_bam, max_retries=max_retries }
-    call qualimap.rnaseq as qualimap_rnaseq { input: bam=picard_sort.sorted_bam, gencode_gtf=gencode_gtf, strand=strand, inferred=ngsderive_strandedness.strandedness, max_retries=max_retries }
-    call htseq.count as htseq_count { input: bam=picard_sort.sorted_bam, gtf=gencode_gtf, strand=strand, inferred=ngsderive_strandedness.strandedness, max_retries=max_retries }
-    call samtools.flagstat as samtools_flagstat { input: bam=picard_sort.sorted_bam, max_retries=max_retries }
+    call htseq.count as htseq_count { input: bam=picard_sort.sorted_bam, gtf=gencode_gtf, provided_strand=provided_strand, inferred_strand=ngsderive_strandedness.strandedness, max_retries=max_retries }
     call md5sum.compute_checksum { input: infile=picard_sort.sorted_bam, max_retries=max_retries }
```

Comment:
We can remove this here, it's going to be computed as the first step in the QC pipeline.

Reply:
@adthrasher 's TSNE workflow relies on gene counts being computed in this workflow. However, it wouldn't be hard to modify that workflow to just run `htseq-count` on its own. Which route should we go? @claymcleod

Reply:
I'm referring to the md5sum here, not htseq count